### PR TITLE
fix/prevent excessive dom size , feat/add tokyo night theme a

### PR DESCRIPTION
**Pull request by rendi12345678**

### CODE REVIEW

**Comment by rendi12345678** [comment]
https://github.com/user-attachments/assets/ae1c0349-b5bc-4c43-a0fb-54358b9d0395

**Comment by rendi12345678** [comment]
https://github.com/user-attachments/assets/1eeb0ce6-eb03-40f1-a939-148910f19242

new language demo

File: src/algorithm.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Algorithm to get total speed target wpm
+
+1. Get target wpm
+2. Get to know how to calculate wpm
+3.
```
Comment (MUYANGGUO) [inline src/algorithm.md:5]: this seems not finished, but not a blocker

**Review by MUYANGGUO — COMMENTED** [review]
LGTM, will merge now

**Review by MUYANGGUO — APPROVED** [review]

**Pull request merged**